cookbook/agent_concepts/knowledge/filters/user_level_filtering.py

```diff
@@ -0,0 +1,54 @@
+from agno.agent import Agent
+from agno.document import Document
+from agno.knowledge.pdf_url import PDFUrlKnowledgeBase
+from agno.vectordb.qdrant import Qdrant
+
+# Collection name
+COLLECTION_NAME = "recipes"
+
+# Initialize Qdrant vector database
+vector_db = Qdrant(collection=COLLECTION_NAME, url="http://localhost:6333")
+
+# Initialize knowledge base
+knowledge_base = PDFUrlKnowledgeBase(
+    vector_db=vector_db,
+)
+
+knowledge_base.load_url(
+    url="https://agno-public.s3.amazonaws.com/recipes/thai_recipes_short.pdf",
+    metadata={"user_id": "user_1", "source": "Thai Cookbook"},
```

Comment:
Lets use better examples. You can load URLs of certain tools/products that you want to ask questions about, or you can load txt about users (some private info like CV) if you want to stick to users.